test/tests/functional/pbs_test_entity_limits.py

```diff
@@ -0,0 +1,626 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2017 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestEntityLimits(TestFunctional):
+
+    """
+    This test suite tests working of queued_jobs_threshold, max_queued,
+    queued_jobs_threshold_res and max_queued_res.
+
+    PBS supports entity limits at queue and server level. And these limits
+    can be applied for a user, group, project or overall.
+    This test suite iterates over all the entities.
+
+    In all there are 28 tests, 14 each for queued_jobs_threshold and
```

Comment:
I see there are about 56 tests, please update this line

Reply:
fixed in commit rc9 https://github.com/PBSPro/pbspro/pull/807/commits/29aa9a33958b81a20f97b36dd8c88c48d6499583